### PR TITLE
#107 - Academy Main Content Section

### PR DESCRIPTION
**Pull request by jhancock532**
### [Link to Ticket On Codebase](https://projects.torchbox.com/projects/contentful-careers-page/tickets/107)

Added the main content section directly to the academies page. I didn't add this as a separate component as it's three divs, this still might be worthwhile to refactor.

### How to Test

[Link to preview site page with component](https://careers-2ngwxkrqp-careers-site.vercel.app/careers/torchbox-academy)

<details>
  <summary>Screenshots</summary>
image
image
image
</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [x] Tests and linting passes.
- [x] Consider updating documentation. If you don't, tell us why.
- [x] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: styles/pages/TorchboxAcademy.module.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.container {
```
Comment (Morsey187) [inline styles/pages/TorchboxAcademy.module.scss:1]: **Suggestion**:  Try to use SCSS varaibles ($site-width, $content--narrow) when possible.

File: styles/pages/TorchboxAcademy.module.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+.textContainer {
```
Comment (Morsey187) [inline styles/pages/TorchboxAcademy.module.scss:6]: **Suggestion**: I would delete this if it's not being used

File: styles/pages/TorchboxAcademy.module.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @include media-query(large) {
+        padding: 0;
+    }
+}
+
+.mainContentContainer {
+    margin: 0 auto $grid * 3 auto;
+    max-width: 784px;
```
Comment (Morsey187) [inline styles/pages/TorchboxAcademy.module.scss:22]: **Suggestion:** not really a suggestion but something to keep in mind.

You could add a local SCSS variable to this file i.e. $extra-text-padding;
`max-width: $content--narrow + $extra-text-padding;`
This one might not be ideal here but can be useful if you have attributes that depends on the same size of another locally, such as when we indent content to the same size as the padding.

Round the padding down and use grid (might not fit the designs but could be better for consistency)
`max-width: $content--narrow + $grid;`

Or
`max-width: $content--narrow + 29px;`

File: styles/pages/TorchboxAcademy.module.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        padding: 0;
+    }
+}
+
+.secondaryTitle {
+    @include font-size(xxxl);
+    font-weight: $weight--extrabold;
+    line-height: 1;
+    margin-bottom: 35px;
+    max-width: 350px;
```
Comment (Morsey187) [inline styles/pages/TorchboxAcademy.module.scss:40]: Might be nice to add this one and 500px to "// Wrappers" inside _variables.scss

**Review by Morsey187 — APPROVED** [review]
LGTM just some minor comments on sizing

File: styles/pages/TorchboxAcademy.module.scss (hunk 1 above)
Comment (jhancock532) [reply styles/pages/TorchboxAcademy.module.scss:1]: Thank you - I've been copying some of this code over from other pages, will update this as well. Might be useful as a utility class...

File: styles/pages/TorchboxAcademy.module.scss (hunk 2 above)
Comment (jhancock532) [reply styles/pages/TorchboxAcademy.module.scss:6]: This is being used on another branch, I saved this file locally so I have the code from a previous MR

File: styles/pages/TorchboxAcademy.module.scss (hunk 3 above)
Comment (jhancock532) [reply styles/pages/TorchboxAcademy.module.scss:22]: Thank you, that's good to know! I will try this out here 👍

File: styles/pages/TorchboxAcademy.module.scss (hunk 1 above)
Comment (Morsey187) [reply styles/pages/TorchboxAcademy.module.scss:1]: Yeah good idea, I think a utility class would work well here and this is the ideal case for one 👍

**Pull request merged**